integration-tests/tests/client/process_blocks.rs

```diff
@@ -1949,8 +1954,23 @@ fn test_block_merkle_proof_with_len(n: NumBlocks) {
             cur_height += 1;
         }
     }
+
     let head = blocks.pop().unwrap();
     let root = head.header().block_merkle_root();
+    // verify that block ordinal to block hash map is correct
```

Comment:
what is "block hash map" ? I thought that we're comparing with ColBlockMerkleTree ?

Reply:
it is [block ordinal to block hash] map. Will rephrase it to make it easier to parse